packages/test/test-end-to-end-tests/src/test/summarizeRefreshLatestSummary.spec.ts

```diff
@@ -53,5 +49,88 @@ describeCompat(
 			await createSummarizer(provider, container, undefined, summaryVersion);
 			await provider.ensureSynchronized();
 		});
+
+		it("The summarizing client will immediately refresh its own summaries", async () => {
+			if (provider.driver.type !== "local") {
+				// skip other drivers as the test would look quite different just to replicate the same behavior
+				return;
+			}
+			const summaryConfig: ISummaryConfiguration = {
+				...DefaultSummaryConfiguration,
+				...{
+					maxTime: 5000 * 12,
+					maxAckWaitTime: 120000,
+					maxOps: 1,
+					initialSummarizerDelayMs: 0,
+				},
+			};
+			const mockLogger = new MockLogger();
+			const testContainerConfig: ITestContainerConfig = {
+				runtimeOptions: {
+					summaryOptions: {
+						summaryConfigOverrides: summaryConfig,
+					},
+				},
+				loaderProps: {
+					logger: mockLogger,
+					configProvider: mockConfigProvider({
+						"Fluid.Summarizer.immediatelyRefreshLatestSummaryAck": true,
+					}),
+				},
+			};
+			const container = await provider.makeTestContainer(testContainerConfig);
+			const container2 = await provider.loadTestContainer(testContainerConfig);
+			const dataObject = (await container.getEntryPoint()) as ITestDataObject;
+			const dataObject2 = (await container2.getEntryPoint()) as ITestDataObject;
+			await waitForContainerConnection(container);
+			await waitForContainerConnection(container2);
+
+			// The first summary needs to be generated otherwise the summarizer will only generate one summary.
```

Comment:
I did not understand what this comment means.

Reply:
Added additional clarifying comments. This is to put the summarizer client in the "right" state for conditions of two summaries occurring simultaneously one after another.